src/main/scripts/org/reaktivity/specification/nukleus/amqp/streams/send.to.client.with.annotations/server.rpt

```diff
@@ -40,7 +40,7 @@ write nukleus:begin.ext ${amqp:beginEx()
 
 connected
 
-write nukleus:data.ext ${amqp:dataEx()
+write nukleus:data.ext ${amqp:matchDataEx()
```

Comment:
This should be `write nukleus:data.ext ${amqp:dataEx()` because matching is a readonly operation.